src/core/Provider.ts

```diff
@@ -1,9 +1,9 @@
-import { createElement, useRef, useDebugValue, FC } from 'react'
+import { FC, createElement, useDebugValue, useRef } from 'react'
```

Comment:
It might be out of the scope of this PR, but as we strictly use `import type` for types, this would be:
```suggestion
import { createElement, useDebugValue, useRef } from 'react'
import type { FC } from 'react'
```